docs/setup/settings.asciidoc

```diff
@@ -145,14 +145,14 @@ By turning this off, only the layers that are configured here will be included.
 `status.allowAnonymous:`:: *Default: false* If authentication is enabled, setting this to `true` allows
 unauthenticated users to access the Kibana server status API and status page.
 
-[[tilemap-settings]] `tilemap.options.attribution:`:: *Default: `"© [Elastic Maps Service](https://www.elastic.co/elastic-maps-service)"`* The map attribution string. Supported on Elastic Cloud Enterprise.
+[[tilemap-settings]] `map.tilemap.options.attribution:`:: *Default: `"© [Elastic Maps Service](https://www.elastic.co/elastic-maps-service)"`* The map attribution string. Supported on Elastic Cloud Enterprise.
```

Comment:
could we move up all the map.tilemap.* settings so they all the map.* settings are together?

Reply:
similar comment for `map.regionmap.*`

Reply:
That makes sense to me. Docs updated!